docs/src/modules/java/pages/views.adoc

```diff
@@ -53,6 +53,8 @@ include::example$java-valueentity-customer-registry/src/main/proto/customer/cust
 <4> The `GetCustomers` method defines the query to retrieve a stream of customers.
 <5> The `(akkaserverless.method).view.query` annotation defines that this method is used as a query of the View.
 
+NOTE: In this sample we use the internal `domain.CustomerState` as the state of the view. This is convenient since it allows automatic updates of the view without any logic but has the draw back that it implicitly makes the `domain.CustomerState` type a part of the public service API. Transforming the state to another type than the incoming update to avoid this can bee seen in <<#event-sourced-entity>>.
```

Comment:
```suggestion
NOTE: In this sample we use the internal `domain.CustomerState` as the state of the view. This is convenient since it allows automatic updates of the view without any logic but has the draw back that it implicitly makes the `domain.CustomerState` type a part of the public service API. Transforming the state to another type than the incoming update to avoid this can be seen in <<#event-sourced-entity>>.
```